frontend/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,62 @@
+import { Column, useTable } from 'react-table';
+import { FC } from 'common/types/types';
+import { ReactTableRenderEntitiesNames } from './common/react-table-render-entities-names';
+import styles from './table/styles.module.scss';
+
+type Props = {
+  columns: Column[];
+  data: unknown[];
+};
+
+const Table: FC<Props> = ({ columns, data }) => {
+  const tableInstance = useTable({
+    columns: columns as Column<Record<string, string>>[],
+    data: data as Record<string, string>[],
+  });
+
+  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
+    tableInstance;
+
+  return (
+    <table {...getTableProps()} className={styles.table}>
+      <thead className={styles['table-head']}>
```

Comment:
```suggestion
      <thead className={styles.tableHead}>
```
use dot notation for all

Reply:
Am I right in idea that I should transform classes names from kebab-case (from .module.scss file) into camelCase inside the table component (and use helper for that transformation). i could change classes in scss file, but git hook needs kebab-case in scss files

Reply:
Write styles in `camelCase`

Reply:
But how to handle this issue than?
![Code_B7TBJTY11r](https://user-images.githubusercontent.com/88377450/183284152-a99813c0-5938-4951-a5f3-6dfa5a4a40c2.png)

Reply:
pull last changes from development

Reply:
I stashed changes in my current task, switched to "development" branch, pulled changes. After I switched to my branch, applied stash and than tried to commit again, problem with git hook left.

Reply:
Should I use that kebab-case notion in table.tsx? When hook is changed I will switch to camelCase notion

Reply:
you should use camelCase in both, scss and tsx